pkg/controller/jobs/pod/pod_controller.go

```diff
@@ -540,14 +540,14 @@ func (p *Pod) Skip() bool {
 
 // podGroupName returns a value of GroupNameLabel for the pod object.
 // Returns an empty string if there's no such label.
-func podGroupName(p corev1.Pod) string {
+func PodGroupName(p *metav1.ObjectMeta) string {
```

Comment:
```suggestion
func podGroupName(p *metav1.ObjectMeta) string {
```
On this way we don't need to make it public.